tests/filecheck/dialects/print/print_basics.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: xdsl-opt %s | filecheck %s
+
+builtin.module {
+    print.println "Hello world!"
+
+    %144 = "arith.constant"() {"value" = 144 : i32} : () -> i32
+    %12 = "arith.constant"() {"value" = 12 : i32} : () -> i32
+
+    print.println "Did you know that {}^2 = {}, and fib({}) = {} too!", %12 : i32, %144 : i32, %12 : i32, %144 : i32
+
+    print.println "Furthermore, {} is the smallest number with 15 divisors", %144 : i32 {"test_thing" = 32 : i32}
+}
+
+// CHECK:      builtin.module {
```

Comment:
Why check for the whole module? You are mainly interested in the different `println`- lines, right?